modules/api/src/main/scala/com/github/scoquelin/arugula/commands/RedisSortedSetAsyncCommands.scala

```diff
@@ -401,4 +423,11 @@ object RedisSortedSetAsyncCommands {
   final case class ScoreWithKeyValue[K, V](score: Double, key: K, value: V)
   final case class ZRange[T](start: T, end: T)
   final case class RangeLimit(offset: Long, count: Long)
+
+  sealed trait SortOrder
```

Comment:
I didn't know what to call this enum. Direction didn't seem quite right to me.

Reply:
`SortPriority` maybe ? Anyway, `SortOrder` works fine too!